graphprot/NeuralNet.py

```diff
@@ -123,7 +123,11 @@ def __init__(self, database, Net,
         self.valid_loss = []
 
     def plot_loss(self, name=''):
-        """Plot the loss of the model."""
+        """Plot the loss of the model
+
+        Args:
+            name (str, optional): name of the output file. Defaults to ''.
```

Comment:
that's the epoch number right ?

Reply:
No the name is not the epoch since the plot is the loss as a function of the epoch.
The plot can be computed after the model is trained, and the name can be any input from the user.

ex:
```
model.train(nepoch=epoch, validate=True)
model.plot_loss('_fold{}_b{}_e{}_t{}_y{}_shuffle_onehot_EGAT'.format(fold, batch_size, epoch, task, target))

```